recipes/hkl/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+test -d m4 || mkdir m4
+gtkdocize || exit 1
+
+export python=$PYTHON
+export PATH=$PREFIX/bin:$PATH
+export PKG_CONFIG_LIBDIR="$PREFIX/lib/pkgconfig"
+export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:$PKG_CONFIG_LIBDIR
+export ACLOCAL_PATH="$PREFIX/share/aclocal"
+aclocal --print-ac-dir
+
+autoreconf -ivf
+
+./configure --disable-gui --enable-introspection=yes --disable-hkl-doc --prefix=$PREFIX 
+# || { cat config.log ; exit 1 ; }
+make -j
```

Comment:
This should be `make -j ${CPU_COUNT}`

Reply:
Why is it bad? Too much stress on the CI?
```
  -j [N], --jobs[=N]          Allow N jobs at once; infinite jobs with no arg.
```

Reply:
It is standard for our recipes

Reply:
This is addressed. Thanks for the hint!